root/usr/libexec/nethserver/dedalo-disable-ethernet-offload

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+hotspot_interface=$(/sbin/e-smith/db networks showjson | jq -r '.[] | select(.props.role=="hotspot") | .name')
```

Comment:
Maybe it's better to use the absolute path for jq

Reply:
The `jq`'s path is resolved correctly, but i will use the absolute path for be sure.